debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java

```diff
@@ -826,20 +826,30 @@ protected void finalizeTransactionCommit(String transactionId, Scn commitScn) {
     }
 
     /**
-     * Check whether the supplied username associated with the specified transaction is excluded.
+     * Check whether the supplied transaction should be skipped based on exclusion rules.
      *
-     * @param transaction the transaction, never {@code null}
-     * @return true if the transaction should be skipped; false if transaction should be emitted
+     * @param transaction the transaction, should not be {@code null}
+     * @return {@code true} if the transaction should be skipped; {@code false} otherwise
      */
-    protected boolean isTransactionUserExcluded(T transaction) {
+    protected boolean isTransactionSkipped(T transaction) {
         if (transaction != null) {
-            if (transaction.getUserName() == null && getTransactionEventCount(transaction) > 0) {
-                LOGGER.debug("Detected transaction with null username {}", transaction);
-                return false;
+            if (!Strings.isNullOrBlank(transaction.getUserName())) {
```

Comment:
While the code is correct, I'd argue that it is bit harder to read/follow given the similarity of the method namess accessing config options and include exclude. I prpose to either insert few empty lines here and there to visually break the code into logice sections or add simple oneline comments. That would the future reader/maintainer.

Reply:
@jpechane can you check the code and see if what I did in 2fa1f3f is sufficient?

Reply:
@Naros That's perfect!